justfile

```diff
@@ -32,5 +32,7 @@ stage1_cpio:
 # Top level target to build all enclave apps and the kernel, and run tests.
 #
 # This is the entry point for Kokoro CI.
-kokoro: all_enclave_apps oak_restricted_kernel_bin stage0_bin stage1_cpio
+kokoro: all_enclave_apps oak_restricted_kernel_bin stage0_bin
     cargo nextest run --all-targets --hide-progress-bar
+
+oak_containers_kokoro: stage1_cpio
```

Comment:
Perhaps let's use `kokoro_` as prefix instead of suffix (both here and in the github actions callback names)